axlearn/common/utils.py

```diff
@@ -1459,3 +1459,30 @@ def validate_contains_paths(x: Nested[Tensor], paths: Sequence[str]):
                 f"Input is expected to contain '{path}'; "
                 f"instead, it contains: '{jax.tree_structure(x)}'."
             ) from e
+
+
+def save_only_these_regex_patterns(*regex_patterns_to_save):
+    """Save only the values that match the regex pattern.
+
+    Args:
+        regexes_to_save: List of regex patterns to save
```

Comment:
Comment on what these regex match?

Reply:
Removed this function in favor of `_save_and_offload_only_these_names_regex`